tests/trunner.nim

```diff
@@ -122,3 +122,9 @@ else: # don't run twice the same test
     block: # mak sure --backend works with `nim r`
       let cmd = fmt"{nim} r --backend:{mode} --hints:off --nimcache:{nimcache} {file}"
       check execCmdEx(cmd) == ("ok3\n", 0)
+
+  block: # some importc tests
+    # issue #14314
+    let file = testsDir / "misc/mimportc.nim"
+    let cmd = fmt"{nim} r -b:cpp --hints:off --nimcache:{nimcache} --warningAsError:ProveInit {file}"
+    check execCmdEx(cmd) == ("witness\n", 0)
```

Comment:
This doesn't need an extra driver, afaict?

Reply:
as is yes I agree, but I intended to grow this test upcoming PR's with things that are awkward to test via the testament spec DSL